src/test/java/cloud/fogbow/ras/core/PluginInstantiatorTest.java

```diff
@@ -20,13 +18,18 @@
 public class PluginInstantiatorTest {
 
     private InteroperabilityPluginInstantiator interoperabilityPluginInstantiator;
+    private static final String fakeCloudName = "default";
```

Comment:
The declaration of constants must follow java language conventions (letters in uppercase separated by underlines). In the specific case, the constant used here is already declared in the auxiliary class 'TestUtils' and is called 'DEFAULT_CLOUD_NAME', just use it and remove this declaration.

Reply:
Fixed.